docs/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM jekyll/jekyll:stable
```

Comment:
Does the local setup still work for you?

Reply:
No, but I also recently switched to Mac 🤔 But changing the platform I still receive some strange errors that I think are just some version upgrade problems 🙈 I tried fixing it for 20 mins without success, but I think I'm gonna postpone this for now...